src/app/containers/MostRead/Canonical/index.jsx

```diff
@@ -33,7 +40,22 @@ const MarginWrapper = styled.div`
   }
 `;
 
-const CanonicalMostRead = ({ endpoint, maxTwoColumns }) => {
+const MaxWidthWrapper = styled.div`
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MAX}) {
+    margin: 0 ${GEL_MARGIN_ABOVE_400PX} ${GEL_SPACING_QUAD};
+  }
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    margin-bottom: ${GEL_SPACING_QUIN};
+  }
+  @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
+    margin: 0 auto ${GEL_SPACING_TRPL};
+    max-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN};
```

Comment:
We apply this max-width above `1008px` for Front Pages. Wondering if we should do the same for Article Pages or UX has requested it like this for this page type.

Reply:
In the UX designs for Articles (and the future 404 error page I believe) the component is allowed to stretch to 1280px. Have confirmed it with Laura as well.

https://app.zeplin.io/project/5baa564c05c2eba0d2b89568/screen/5cee707ca2d3301e40703647